pom.xml

```diff
@@ -4,11 +4,11 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>org.tensorflow</groupId>
-  <artifactId>parent</artifactId>
+  <artifactId>tensorflow-java</artifactId>
   <version>2.0.0-SNAPSHOT</version>
```

Comment:
It has been suggested in our last call that our new artifacts should start from the beginning, in alpha-mode, as we won't guarantee complete stability for a while. I like this idea as well, what do you think?

Therefore, it would mean that `2.0.0-SNAPSHOT` be renamed to `0.1.0-SNAPSHOT` I guess?

Reply:
Yes, something like that. Do you want to do it as part of this PR?

Reply:
yeah, why not 👍